source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -206,4 +206,4 @@ the ``replaceOne()`` method, it replaces the first result.
 
     The ``replaceOne()`` method cannot make changes to a document that
     violate unique index constraints on the collection. See the MongoDB
-    server manual for more information on :manual:`unique indexes </core/index-unique/>`.
+    Server manual for more information on :manual:`unique indexes </core/index-unique/>`.
```

Comment:
[suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".